docs/source/getting-started/installation.rst

```diff
@@ -575,6 +576,41 @@ In the prompt, enter your Nebius Access Key ID and Secret Access Key (see `instr
   aws configure set region <REGION> --profile nebius
   aws configure set endpoint_url <ENDPOINT>  --profile nebius
 
+CoreWeave
+~~~~~~~~~~~~~~~~~~
+
+`CoreWeave <https://www.coreweave.com/>`__ provides cloud infrastructure with a focus on object storage. SkyPilot supports CoreWeave Object Storage, which is S3-compatible.
+
+To configure CoreWeave Object Storage access, you can use AWS CLI commands:
+
+.. code-block:: shell
+
+  # Configure your CoreWeave credentials
+  aws configure --profile coreweave
+
+In the prompt, enter your CoreWeave Access Key ID and Secret Access Key. Select your preferred region and :code:`json` for the default output format.
+
+.. code-block:: text
+
+  AWS Access Key ID [None]: <your_access_key_id>
+  AWS Secret Access Key [None]: <your_secret_access_key>
+  Default region name [None]: <REGION>
+  Default output format [None]: json
+
+Then set the endpoint URL and S3 addressing style:
+
+.. code-block:: shell
+
+  aws configure set endpoint_url https://cwlota.com --profile coreweave
```

Comment:
Currently LOTA only supports `http` and not `https`, also it might be worth a note that cwlota.com only works inside of CKS clusters running in CoreWeave itself. If you're outside of a CoreWeave CKS cluster you should use https://cwobject.com